### ISSUE TITLE
Mode carrousel cannot change radio app UIS7862

### ISSUE THREAD

**Issue by mvrab** [hvdwolf/FytHWOneKey#9]
Using v1.4.0 on my Navifly/Mekede UIS7862, the mode carrousel only changes between radio and the app configured for "BT Phone option" in FytHWOneKey.  All other apps configured in the carrousel are ignored, it only switches between radio and the configured BT Phone option.

If I had to guess what is wrong, it would be that the carrousel now also includes the radio app, which FytHWOneKey does not have a configuration setting for.  FytHWOneKey does come up as an option when switching, but even when set as default the stock Radio still opens.  When the radio opens the carrousel resets its position back to the beginning and the cycle of configured "BT Phone" to Radio repeats.

**Comment by hvdwolf** [comment]
Hi,

I already mentioned that in this XDA post: https://forum.xda-developers.com/t/teyes-cc3.4189041/page-40#post-85131585
The SRC/MOD key is really very troublesome.
And it seems that the default activity for the radio app has changed on the uis7862 compared to the previous sofia 3GR, PX5 and sc9853i. 
I do not knw whether I can fix this and certainly not when.

**Comment by ukiews** [comment]
Same issue here with the Teyes CC3 unit. Wonder if having a rooted device can help overcome this?

**Comment by hvdwolf** [comment]
No. Rooting has nothing to do with it. It has to do with a different program activity that needs to be called.
It might require a statement like "if 7862 then ..... else ....", but I don't have time for it and don't want to spend time on it right now. I am not using my own program anymore as I don't have hardware buttons anymore on my unit.
